extensions/amp-ad/0.1/amp-ad-custom.js

```diff
@@ -96,17 +96,19 @@ export class AmpAdCustom extends AMP.BaseElement {
             null;
       }
       // Set UI state
-      this.uiHandler.setDisplayState(
-        templateData !== null && typeof templateData == 'object' ?
-        AdDisplayState.LOADED_RENDER_START : AdDisplayState.LOADED_NO_CONTENT);
-      templatesFor(this.win).findAndRenderTemplate(element, templateData)
-        .then(renderedElement => {
-        // Get here when the template has been rendered
-        // Clear out the template and replace it by the rendered version
-          removeChildren(element);
-          element.appendChild(renderedElement);
-          return this;
-        });
+      if (templateData !== null && typeof templateData == 'object') {
+        this.uiHandler.setDisplayState(AdDisplayState.LOADED_RENDER_START);
+        templatesFor(this.win).findAndRenderTemplate(element, templateData)
+          .then(renderedElement => {
+          // Get here when the template has been rendered
+          // Clear out the template and replace it by the rendered version
+            removeChildren(element);
+            element.appendChild(renderedElement);
+            return this;
```

Comment:
I'm sure it was requested during a code review at some point, but I can't find it in the comment trail, I've done much head-scratching and I can't remember why. The code appears to work fine without it. Presumably we should take it out - unless @lannka or someone else can remember why it's there! Let me know, or feel free to delete line 107 before merging.